Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -294,12 +366,19 @@ final class DetailingCodebaseViewController: BaseViewController {
             $0.height.equalTo(80)
         }
         
-        view.addSubview(manitoOpenButton)
-        manitoOpenButton.snp.makeConstraints {
+        view.addSubview(manitoOpenButtonShadowView)
+        manitoOpenButtonShadowView.snp.makeConstraints {
             $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(7)
             $0.centerX.equalToSuperview()
+            $0.width.equalTo(UIScreen.main.bounds.size.width - 40)
+            $0.height.equalTo(60.0)
```

Comment:
p3;

```suggestion
            $0.leading.trailing.equalToSuperview().inset(Size.leadingTrailingPadding)
            $0.height.equalTo(60)
```

좌우 값에서 20씩 띄어서 설정해줘도 될 것 같아요 ! 이 방식이 저희가 여태 레이아웃을 잡은 방식이기도 하구요 !

Reply:
> p3;
> 
> 좌우 값에서 20씩 띄어서 설정해줘도 될 것 같아요 ! 이 방식이 저희가 여태 레이아웃을 잡은 방식이기도 하구요 !

1afb7e6 감사합니다~